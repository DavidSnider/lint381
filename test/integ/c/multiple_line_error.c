// Example of an error whose tokens span multiple lines.
if (foo ==
NULL)
