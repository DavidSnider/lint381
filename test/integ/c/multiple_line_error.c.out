multiple_line_error.c:2:9: error: Comparison to NULL should be avoided
if (foo ==
        ^^
